### ISSUE TITLE
how to train the model using weights obtained from pretraining on the imagenet dataset 

### ISSUE THREAD

**Issue by hajungong007** [matterport/Mask_RCNN#977]
hi， **I initialize backbone (resnet101 or resnext101 or resnet50) using weights obtained from pretraining on the imagenet dataset, but the loss declined slowly.** I choose to train the network for 160k steps on the MS COCO 2014 train dataset with a batch size of 2 on a single GPU machine.The training consists of three stages each lasting for 40k, 80k, 40k steps respectively: in the first stage only the Mask R-CNN branches and not the ResNet backbone are trained. Next, the prediction heads and parts of the backbone (starting at layer 4) are optimized. Finally, in the third stage, the entire model (backbone and heads) is trained together. For the first two training stages we use a learning rate of 0.001 and for the last one a decreased learning rate of 0.001/10. The optimization is done by SGD with momentum set to 0.9 and weight decay set to 0.0001.

**Comment by fastlater** [comment]
So what is your question? I just see a copy and paste of the implementation details of the paper called: "Faster training of mask-rcnn by focusing on instance boundaries".
